spec/requests/providers/confirm_non_means_tested_applications_controller_spec.rb

```diff
@@ -114,7 +114,7 @@
 
       it "updates application statre and redirects to the merits task list", :aggregate_failures do
```

Comment:
The description still says a specific page

Reply:
👍